ci/pingcap_tidb_operator_build_kind.groovy

```diff
@@ -190,9 +227,6 @@ def call(BUILD_BRANCH, CREDENTIALS_ID, CODECOV_CREDENTIALS_ID) {
 		builds["E2E v1.12.10"] = {
 			build("${MIRRORS} IMAGE_TAG=${GITHASH} SKIP_BUILD=y GINKGO_NODES=8 KUBE_VERSION=v1.12.10 REPORT_DIR=\$(pwd)/artifacts REPORT_PREFIX=v1.12.10_ ./hack/e2e.sh -- --preload-images --ginkgo.skip='\\[Serial\\]'", artifacts)
 		}
-		builds["E2E v1.16.4"] = {
```

Comment:
remove k8 1.16?

Reply:
yes, reduce resource needed for our CI jobs.
if we test against 1.16 on PR, why not 1.15, 1.14, ...
before we have enough resources to use, we can test only the minimum and the latest versions on PR.
of cause, we can have a job to test against all versions on branch serially.

Reply:
xref: https://github.com/pingcap/tidb-operator/issues/1616